tests/vmcheck/test-kernel-args.sh

```diff
@@ -151,6 +151,22 @@ assert_file_has_content_literal kargs.txt 'PACKAGE=TEST'
 assert_file_has_content_literal kargs.txt 'PACKAGE2=TEST2'
 echo "ok kargs display with multiple operations"
 
+# Test for rpm-ostree kargs append-if-missing and delete-if-present
+vm_rpmostree kargs --append-if-missing=PACKAGE3=TEST3
+vm_rpmostree kargs > kargs.txt
+assert_file_has_content_literal kargs.txt 'PACKAGE3=TEST3'
+vm_rpmostree kargs --append-if-missing=PACKAGE3=TEST3
+vm_rpmostree kargs > kargs.txt
+assert_file_has_content_literal kargs.txt 'PACKAGE3=TEST3'
```

Comment:
A better test here is verifying that the new output matches the old output (i.e. that nothing was changed). So e.g. you can output to a new file, and then use `diff`.